Sources/CoreMetrics/Metrics.swift

```diff
@@ -395,8 +400,9 @@ public protocol MetricsFactory {
     ///
     /// - parameters:
     ///     - label: The label for the `TimerHandler`.
+    ///     - storageUnit: The unit to store the time in.
     ///     - dimensions: The dimensions for the `TimerHandler`.
-    func makeTimer(label: String, dimensions: [(String, String)]) -> TimerHandler
+    func makeTimer(label: String, storageUnit: TimeUnit, dimensions: [(String, String)]) -> TimerHandler
```

Comment:
=> add default to keep API more backwards compatible
=> storageUnit => preferredDisplayUnit
=> change the description in the API doc to say: "displayUnit : A hint to the backend responsible for presenting the data of the preferred display unit. This is not guaranteed to be supported by all backends."